python-checks/src/main/java/org/sonar/python/checks/IgnoredSystemExitCheck.java

```diff
@@ -49,13 +53,15 @@ public class IgnoredSystemExitCheck extends PythonSubscriptionCheck {
 
   private static final String SYSTEM_EXIT_FUNCTION_NAME = "sys.exit";
   private static final String SYSTEM_EXC_INFO_NAME = "sys.exc_info";
+  public static final String NOT_RAISED_CAUGHT_EXCEPTION_QUICK_FIX_MESSAGE = "Replace \"pass\" statement with \"raise\"";
```

Comment:
I'd suggest to use `Propagate the exception` as a message.

Reply:
done